content/en/docs/concepts/architecture/nodes.md

```diff
@@ -93,7 +93,7 @@ For self-registration, the kubelet is started with the following options:
   {{< glossary_tooltip text="taints" term_id="taint" >}} (comma separated `<key>=<value>:<effect>`).
 
   No-op if `register-node` is false.
-- `--node-ip` - IP address of the node.
+- `--node-ip` - IP address of the node, or comma-separated pair of IP addresses in [a dual-stack cluster](/docs/concepts/services-networking/dual-stack/#configure-ipv4-ipv6-dual-stack).
 - `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} to add when registering the node
   in the cluster (see label restrictions enforced by the
   [NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)).
```

Comment:
We might want to mark the other command line arguments that are optional as optional, so that readers don't draw the wrong inference.

Reply:
Did you want me to do that as part of this PR?

Reply:
Nice to have, but not required here.